tools/triage/ticket_search/settings.py

```diff
@@ -0,0 +1,17 @@
+"""Constants that are common to all triage tools."""
+JIRA_SERVER = "https://issues.redhat.com"
+TRIAGE_TICKET_SEARCH_PROJECT = "AITRIAGE"
+TRIAGE_TICKET_SEARCH_COMPONENT = "Cloud-Triage"
```

Comment:
lots of it already exists in other places

Reply:
done apart from "Cloud-Triage" for which we do not presently have a const